resources/lang/en/siwecos.php

```diff
@@ -13,8 +13,6 @@
   |
   */
 
-  'REPORT_FOR' => "Report for :domain",
```

Comment:
@Skeeve kann das wirklich gelöscht werden?

Ich bin der Meinung, dass dies für die Report-Ansicht benötigt wurde.

Reply:
Da stimmt was nicht. Bei mir ist das noch vorhanden. Ich prüfe.

Reply:
:D Alles Gut! In der Alten Datei ist das 2 Mal drin.

Bedenke: Ich generiere die Dateien per Script. Dadurch kommt es zu Änderungen in der Reihenfolge, wenn man sie manuell angefaßt hat. Darum steht auch ganz oben in der Datei:

  | This is a generated file.
  |
  | Any manual change will be lost.